.github/workflows/i18n-string-extract-master.yml

```diff
@@ -0,0 +1,35 @@
+name: Extract and upload i18n strings
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  extract-and-upload-i18n-strings:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up environment
+      uses: ./.github/actions/setup
+    - name: Configure i18n client
+      run: |
+        pip install wlc
+        mkdir -p ~/.config
+        cat <<EOF >~/.config/weblate
+        [weblate]
+        url = https://hosted.weblate.org/api/
+        key = ${{ secrets.WEBLATE_API_KEY_CI_STRINGS }}
+        translation = actualbudget/actual/en
+        EOF
+        chmod 600 ~/.config/weblate
```

Comment:
It doesn't seem to work directly, since variables don't seem to be shared between jobs. Any ideas?

Reply:
So that entire variable is apparently a hallucination; however, I got it working with another route. Can you please review c68d90b9e68d386673dc7161fcca93b00383cbc6?

Reply:
bump

Reply:
@coderabbitai